### PR TITLE
cgu: added TestCguListInAllNamespaces unit tests

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/cgu/cgulist.go [hunk 1]
```diff
@@ -5,17 +5,20 @@ import (
...
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
-	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // ListInAllNamespaces returns a cluster-wide cgu inventory.
-func ListInAllNamespaces(apiClient *clients.Settings, options ...runtimeclient.ListOptions) ([]*CguBuilder, error) {
-	logMessage := string("Listing CGUS in all namespaces ")
+func ListInAllNamespaces(apiClient *clients.Settings, options ...metav1.ListOptions) ([]*CguBuilder, error) {
+	logMessage := "Listing CGUS in all namespaces "
```
Comment (achuzhoy) [inline pkg/cgu/cgulist.go:14]: Do we need the extra space after namespaces?

File: pkg/cgu/cgulist_test.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+func TestCguListInAllNamespaces(t *testing.T) {
+	testCases := []struct {
+		testCGU       []*CguBuilder
+		listOptions   []metav1.ListOptions
+		expectedError error
+		client        bool
+	}{
+		{
```
Comment (achuzhoy) [inline pkg/cgu/cgulist_test.go:19]: Do we want to provide default values explicitly (for example listOptions here?

Comment (trewest) [reply pkg/cgu/cgulist_test.go:19]: Agreed

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**